.github/workflows/smoke-test.yml

```
@@ -0,0 +1,83 @@
+name: Smoke test
```

WDYT about renaming this file so that we know it's related to Botkube Cloud?
e.g.:
```suggestion
name: Botkube Cloud Prod E2E test
```

and the file could be named e.g. `prod-e2e-test.yaml` or something like this. WDYT?